po/nb.po

```diff
@@ -109,7 +109,7 @@ msgstr ""
 #. TRANSLATORS: 'Name <email@domain.com>' or 'Name https://website.example'
 #: src/exm-application.c:112
 msgid "translator-credits"
-msgstr "Kreditering av oversettere"
+msgstr "kreditering av oversettere"
```

Comment:
Ah actually, just noticed this. This should be substituted with your name so you are credited in the about dialog whenever someone uses the translation. So for example, you might put:

`John Doe <email@website.com>`

If you don't want to provide an email, then simply `Firstname Lastname` is fine.